### PR TITLE
Make the comments of phi failure detector clearer

### PR DESCRIPTION
**Pull request by baymaxhuang**
See [issue 405](https://github.com/atomix/atomix/issues/405) for more details.

### CODE REVIEW

File: cluster/src/main/java/io/atomix/cluster/impl/PhiAccrualFailureDetector.java [hunk 1]
```diff
@@ -22,7 +22,9 @@
 /**
  * Phi Accrual failure detector.
  * <p>
- * Based on a paper titled: "The φ Accrual Failure Detector" by Hayashibara, et al.
+ * A modified version based on a paper titled:
+ * "The φ Accrual Failure Detector" by Hayashibara, et al.
+ * See <a href="https://github.com/atomix/atomix/issues/405">issue 405</a> for more details.
```
Comment (johnou) [inline cluster/src/main/java/io/atomix/cluster/impl/PhiAccrualFailureDetector.java]: I don't think linking to the Github issue is necessary.

File: cluster/src/main/java/io/atomix/cluster/impl/PhiAccrualFailureDetector.java [hunk 2]
```diff
@@ -126,6 +128,7 @@ public double phi() {
    * @param currentTime   the current time
    * @return phi
    */
+  // In this version, phi value is calculated based on an exponential distribution.
```
Comment (johnou) [inline cluster/src/main/java/io/atomix/cluster/impl/PhiAccrualFailureDetector.java]: should be part of the javadoc.

File: cluster/src/main/java/io/atomix/cluster/impl/PhiAccrualFailureDetector.java (hunk 1 above)
Comment (kuujo) [reply cluster/src/main/java/io/atomix/cluster/impl/PhiAccrualFailureDetector.java]: Concur

**Comment by johnou** [comment]
@baymaxhuang ping

**Comment by baymaxhuang** [comment]
@coveralls I have updated it with a new patch set.

**Pull request merged**